service_load_balancing/launch_type_ec2.tf

```diff
@@ -0,0 +1,79 @@
+// NOTE: var.launch_type = "EC2" if you want use following
+
+resource "aws_ecs_service" "main" {
+  count                              = "${ var.launch_type == "EC2" ? 1 : 0 }"
+
+  name                               = "${var.name}"
+  cluster                            = "${var.cluster_id}"
+  launch_type                        = "${var.launch_type}"
+  task_definition                    = "${aws_ecs_task_definition.container.arn}"
+  iam_role                           = "${aws_iam_role.ecs_service.arn}"
+  # Note: To prevent a race condition during service deletion,
+  #       make sure to set depends_on to the related aws_iam_role_policy;
+  #       otherwise, the policy may be destroyed too soon and the ECS service will then get stuck in the DRAINING state.
+  depends_on                         = ["aws_iam_role_policy.ecs_service"]
+
+  # As below is can be running in a service during a deployment
+  desired_count                      = "${var.desired_count}"
+  deployment_maximum_percent         = "${var.deployment_maximum_percent}"
+  deployment_minimum_healthy_percent = "${var.deployment_minimum_healthy_percent}"
+
+  placement_strategy {
+    type  = "${var.placement_strategy_type}"
+    field = "${var.placement_strategy_field}"
+  }
```

Comment:
[NOTE] `aws_ecs_service.placement_strategy` is deprecated
https://www.terraform.io/docs/providers/aws/r/ecs_service.html#placement_strategy

Reply:
thx your feedback, I'll fixup but another pulling requests as soon :)